src/components/MatchSelector.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Selector from './Selector'
+
+class MatchSelector extends React.Component {
+
+    render() {
+        const n = 5;
+        var buttons = []
+        for (let i = 0; i < n; i++) {
+            buttons.push(<Selector type="Match" options={["Player1", "Player2"]}/>);
+            }
+        
+        return (
+            <div>
+               {buttons}
+            </div>
+        )
```

Comment:
```
const defaultOptions = ["player 1", "player 2"];
const options = (new Array(5)).fill(defaultOptions);
return (
<div>
{options.map((options) => <Selector type="Math" options={options}} />)}
</div>
)
```

Reply:
https://fr.reactjs.org/docs/lists-and-keys.html pour les exemples avec les listes. La bonne pratique c'est de ne pas utiliser de boucle for quand ce n'est pas nécessaire :)

Reply:
done